### ISSUE TITLE
Request for faster granular adjustment or manual entry for some menu options

### ISSUE THREAD

**Issue by n1ghttrap** [libretro/RetroArch#10157]
The video scaling and position values in particular can't be adjusted nearly fast enough, and it would be nice to do this from within Retroarch to see the positioning/scaling in real time.  It would be nice to be able to manually enter values for these and/or hold down a key to adjust these faster.  That is all.  Thank you.

**Comment by Ryunam** [comment]
I agree, it would also be particularly useful to have a Zoom option that allows to easily scale the image and zoom-in or zoom-out while keeping the aspect ratio intact.
